tests/compile.sh

```diff
@@ -109,51 +89,40 @@ fi
 # this line can be removed once FMS becomes a pre-installed library
 mkdir -p $PATHTR/FV3/ccpp/include
 
-CMAKE_FLAGS="${CMAKE_FLAGS} -DMPI=ON"
+CMAKE_FLAGS+=" -DMPI=ON"
 
-if [[ "${MAKE_OPT}" == *"DEBUG=Y"* ]]; then
-  CMAKE_FLAGS="${CMAKE_FLAGS} -DCMAKE_BUILD_TYPE=Debug"
-elif [[ "${MAKE_OPT}" == *"REPRO=Y"* ]]; then
-  CMAKE_FLAGS="${CMAKE_FLAGS} -DCMAKE_BUILD_TYPE=Bitforbit"
+if [[ "${MAKE_OPT}" == *"-DDEBUG=ON"* ]]; then
+  CMAKE_FLAGS+=" -DCMAKE_BUILD_TYPE=Debug"
+elif [[ "${MAKE_OPT}" == *"-DREPRO=ON"* ]]; then
+  CMAKE_FLAGS+=" -DCMAKE_BUILD_TYPE=Bitforbit"
 else
-  CMAKE_FLAGS="${CMAKE_FLAGS} -DCMAKE_BUILD_TYPE=Release"
+  CMAKE_FLAGS+=" -DCMAKE_BUILD_TYPE=Release"
   if [[ "${MACHINE_ID}" == "jet.intel" ]]; then
-    CMAKE_FLAGS="${CMAKE_FLAGS} -DSIMDMULTIARCH=ON"
+    CMAKE_FLAGS+=" -DSIMDMULTIARCH=ON"
   fi
 fi
 
   # Check if suites argument is provided or not
 set +ex
-TEST=$( echo $MAKE_OPT | grep -e "SUITES=" )
+TEST=$( echo $MAKE_OPT | grep -e "-DCCPP_SUITES=" )
 if [[ $? -eq 0 ]]; then
-  CCPP_SUITES=$( echo $MAKE_OPT | sed 's/.*SUITES=//' | sed 's/ .*//' )
-  echo "Compiling suites ${CCPP_SUITES}"
+  SUITES=$( echo $MAKE_OPT | sed 's/.*-DCCPP_SUITES=//' | sed 's/ .*//' )
+  echo "Compiling suites ${SUITES}"
 fi
 set -ex
```

Comment:
I do not see where `SUITES` is used, other than in `echo`.

Reply:
I don't see this posing an issue.